### PR TITLE
add link about AD config for BitLocker

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8621

### CODE REVIEW

**Comment by nam31** [comment]
@JohanFreelancer9, can you please do the copy editor review for this PR. Thank you

File: windows/security/information-protection/bitlocker/bitlocker-group-policy-settings.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ Most of the BitLocker Group Policy settings are applied when BitLocker is initia
 If multiple changes are necessary to bring the drive into compliance, you must suspend BitLocker protection, make the necessary changes, and then resume protection. This situation could occur, for example, if a removable drive was initially configured to be unlocked with a password and then Group
 Policy settings are changed to disallow passwords and require smart cards. In this situation, you need to suspend BitLocker protection by using the [Manage-bde](/windows-server/administration/windows-commands/manage-bde) command-line tool, delete the password unlock method, and add the smart card method. After this is complete, BitLocker is compliant with the Group Policy setting and BitLocker protection on the drive can be resumed.
 
+> [!NOTE]
+> For more details about Active Directory configuration related to BitLocker enablement, please check [this article](https://docs.microsoft.com/en-us/windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker).
```
Comment (illfated) [inline windows/security/information-protection/bitlocker/bitlocker-group-policy-settings.md:41]: The target page is language agnostic, handles other languages quite OK, so we don't need to specify /en-us/ here:
```suggestion
> For more details about Active Directory configuration related to BitLocker enablement, please check [this article](https://docs.microsoft.com/windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker).
```

Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-group-policy-settings.md:41]:
```suggestion
> For more details about Active Directory configuration related to BitLocker enablement, please see [Set up MDT for BitLocker](https://docs.microsoft.com/windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/information-protection/bitlocker/bitlocker-group-policy-settings.md (hunk 1 above)
Comment (illfated) [reply windows/security/information-protection/bitlocker/bitlocker-group-policy-settings.md:41]: Ah, right I notice a better suggestion has been made below. I defer to the more experienced suggestion.

**Review by illfated — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**